### PR TITLE
https://issues.redhat.com/browse/ACM-12376 Validate kubeconfig docs

### PR DESCRIPTION
**Pull request by oafischer**
For 2.11 and 2.10

### CODE REVIEW

File: clusters/cluster_lifecycle/scale_node_ocp.adoc [hunk 1]
```diff
@@ -1,30 +1,110 @@
...
+
+. Create an updated `kubeconfig` file by copying your original file. Run the following command and replace `<new_kubeconfig_name>` with the name of your new `kubeconfig` file:
+
++
+[source,bash]
+----
+cp original-kubeconfig <new_kubeconfig_name>
+----
+
+. Append new certificates to the decoded pem by running the following command:
```
Comment (carbonin) [inline clusters/cluster_lifecycle/scale_node_ocp.adoc:57]: I feel like we haven't explained what the problem is so talking about "new certificates" here seems confusing.

Maybe we can mention somewhere at the top that this procedure is for updating the kubeconfig with new CA certs for when the API certs in the spoke cluster have been changed. And also we should mention a prereq is to have the new CA certs required to trust the spoke cluster API

File: clusters/cluster_lifecycle/scale_node_ocp.adoc [hunk 2]
```diff
@@ -1,30 +1,110 @@
...
+
+. Check if the new `kubeconfig` is valid by querying the API with the new `kubeconfig`. Run the following command. Replace `<new_kubeconfig_name>` with the name of your new `kubeconfig` file:
+
++
+[source,bash]
+----
+KUBECONFIG=<new_kubeconfig_name> oc get nodes
+----
++
+If the `kubeconfig` is valid, the workers report a `Ready` status.
```
Comment (carbonin) [inline clusters/cluster_lifecycle/scale_node_ocp.adoc]: The status of the nodes is not relevant. I the user gets any valid response then the kubeconfig is valid.

**Comment by oafischer** [comment]
Made some updates based on your comments @carbonin , feel free to take another look.

File: clusters/cluster_lifecycle/scale_node_ocp.adoc [hunk 3]
```diff
@@ -1,30 +1,120 @@
...
+
+. Check if the new `kubeconfig` is valid by querying the API with the new `kubeconfig`. Run the following command. Replace `<new_kubeconfig_name>` with the name of your new `kubeconfig` file:
+
++
+[source,bash]
+----
+KUBECONFIG=<new_kubeconfig_name> oc get nodes
+----
++
+If you receive an output, the `kubeconfig` is valid.
```
Comment (carbonin) [inline clusters/cluster_lifecycle/scale_node_ocp.adoc]: Maybe "If you receive successful output"? An error is output after all :shrug:

**Review by carbonin — COMMENTED** [review]
Looking much better, just one small comment.

File: clusters/cluster_lifecycle/scale_node_ocp.adoc (hunk 3 above)
Comment (oafischer) [reply clusters/cluster_lifecycle/scale_node_ocp.adoc]: Done

File: clusters/cluster_lifecycle/scale_node_ocp.adoc [hunk 4]
```diff
@@ -1,30 +1,120 @@
...
+
+[#add-nodes-ocp-infra-env-prereqs]
+== Prerequisite
+
+You must have the new CA certificates required to trust the managed cluster API.
+
+[#add-nodes-ocp-infra-env-kubeconfig]
+== Creating a valid `kubeconfig`
+
+Before adding production environment worker nodes to {ocp-short} clusters, you must check if the you have a valid `kubeconfig`. If the API certificates in your managed cluster have changed, complete the following steps to update the `kubeconfig` with new CA certificates:
```
Comment (swopebe) [inline clusters/cluster_lifecycle/scale_node_ocp.adoc]: I would separate this. The IF statement is very valuable and needs to be right before the procedure.

_have changed_ becomes simply _changed_

File: clusters/cluster_lifecycle/scale_node_ocp.adoc [hunk 5]
```diff
@@ -1,30 +1,120 @@
...
+* <<add-nodes-ocp-infra-env-kubeconfig,Creating a valid `kubeconfig`>>
+* <<add-nodes-ocp-infra-env-kubeconfig-steps,Adding worker nodes>>
+
+[#add-nodes-ocp-infra-env-prereqs]
+== Prerequisite
+
+You must have the new CA certificates required to trust the managed cluster API.
+
+[#add-nodes-ocp-infra-env-kubeconfig]
+== Creating a valid `kubeconfig`
```
Comment (swopebe) [inline clusters/cluster_lifecycle/scale_node_ocp.adoc:18]: Can we shorten these anchors? **add-nodes-ocp-cim**

**add-nodes-cim-prereqs**? 

Also, they should more match the title of the section: 
**add-nodes-create-config**
**add-nodes-procedure**

**Review by swopebe — APPROVED** [review]

**Pull request merged**